particles/Services/ImageClassifier.recipes

```diff
@@ -0,0 +1,5 @@
+import 'particles/ImageClassifier.particle'
+recipe ImageClassifier
+  ImageClassifier
+  description `image classification using modular model`
+
```

Comment:
Keep in mind that the `particles/Services` folder was intended only to provide Particles that specifically test/demo Services. Real particles that do real work should live somewhere else (I don't know if you consider these demo particles or not). The fact that such Particles use Services is nobody's business really.